include/cuter_metric_definitions.hrl

```diff
@@ -0,0 +1,10 @@
+%% -*- erlang-indent-level: 2 -*-
```

Comment:
I think we do not need the `_definitions` part in the name of this header file...

Reply:
Done.